pkg/asset/cluster/tfvars.go

```diff
@@ -62,6 +66,7 @@ import (
 	"github.com/openshift/installer/pkg/types/openstack"
 	"github.com/openshift/installer/pkg/types/ovirt"
 	"github.com/openshift/installer/pkg/types/powervs"
+	powervstypes "github.com/openshift/installer/pkg/types/powervs"
```

Comment:
Linter is still complaining about this double import.

Reply:
There was a recent commit adding linters to the config, so you might not have it if you haven't rebased on top of master recently

Reply:
Instead of removing the previous import, I'd suggest removing the one you're introducing and replacing `powervstypes` by `powervs` in your code. The build is broken right now.